clang/test/CIR/CodeGen/struct.c

```diff
@@ -12,11 +12,18 @@ struct Foo {
   struct Bar z;
 };
 
+// Recursive type
+typedef struct Node {
+     struct Node* next;
```

Comment:
Please use only 2 spaces for indentation here and in all of your upcoming PRs. Thanks!